src/networkpolicy/networkPolicy.go

```diff
@@ -1668,7 +1686,24 @@ func mergeEgressPolicies(existPolicy types.KnoxNetworkPolicy, policies []types.K
 						}
 					}
 				}
+			} else if len(newEgress.ToCIDRs) > 0 && len(newEgress.ToPorts) > 0 {
```

Comment:
This is duplicate code. Please keep an API for CIDR matching and call from both the ToCIDRs and FromCIDRs condition.